src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -128,10 +131,14 @@ public static void assertCommandFailure(Command command, Model actualModel, Stri
         // only do so by copying its components.
         AddressBook expectedAddressBook = new AddressBook(actualModel.getAddressBook());
         List<Supplier> expectedFilteredList = new ArrayList<>(actualModel.getFilteredSupplierList());
+        Inventory expectedInventory = new Inventory(actualModel.getInventory());
+        List<Good> expectedGoodFilteredList = new ArrayList<>(actualModel.getFilteredGoodList());
 
         assertThrows(CommandException.class, expectedMessage, () -> command.execute(actualModel));
         assertEquals(expectedAddressBook, actualModel.getAddressBook());
+        assertEquals(expectedInventory, actualModel.getInventory());
         assertEquals(expectedFilteredList, actualModel.getFilteredSupplierList());
+        assertEquals(expectedGoodFilteredList, actualModel.getFilteredGoodList());
```

Comment:
You mean all this time, this is not updated?

Reply:
Ok, time to panic

Reply:
Not adding transaction history comparison?

Reply:
In the implementation of the inventory, there is no new command added. so this method was not called since there is no test on buy/sell command back then. i will add transaction history comparison in.

Reply:
I hope the test don't fail after that